redbot/cogs/audio/core/commands/player.py

```diff
@@ -62,7 +65,7 @@ async def command_play(self, ctx: commands.Context, *, query: str):
             )
         if not self._player_check(ctx):
             if self.lavalink_connection_aborted:
-                msg = _("Connection to Lavalink has failed")
+                msg = _("Connection to Lavalink has failed.")
```

Comment:
```suggestion
                msg = _("Connection to Lavalink has failed")
```
This is the title, to keep the UI for audio consistent there's a few rules to follow.

Title messages shouldn't have ponctuation if there is a description, given that the description here is conditional on whos running it, now im questioning this suggestion ... ill let @aikaterna chime in but I would lean toward not having this dot here.